src/main/java/net/snowflake/client/core/SFResultSetMetaData.java

```diff
@@ -477,4 +478,17 @@ public List<Boolean> getIsAutoIncrementList() {
   public List<SnowflakeColumnMetadata> getColumnMetadata() {
     return columnMetadata;
   }
+
+  @SnowflakeJdbcInternalApi
```

Comment:
Should it be marked as internal?

Reply:
It could because this class isn't public for customer at all